src/alsa.c

```diff
@@ -476,10 +475,11 @@ static int convert_prange(long val, long min, long max) {
  * Adjusts the current volume and sends a notification (if enabled).
  *
  * @param vol new volume value
+ * @param dir select direction
```

Comment:
There are no enums in the project so far, and I was trying to follow the coding standards of pnmixer code. IMHO an integer is good enough. In the used ALSA API call there is also an integer, not enum, so there is some kind of consistency here.